apps/system/style/notifications/notifications.css

```diff
@@ -146,15 +147,20 @@
 
 #notification-bar button {
   flex: initial;
-  width: auto;
+  width: 9.5rem;
   height: 4.5rem;
-  padding: 0;
+  padding: 0 0 0 1.5rem;
   border: 0;
-  background: none;
-  color: #008eab;
+  border-radius: 0;
+  color: #000000;
+  background-color: none;
```

Comment:
we actually want to use `background:none` here to override all the default background styling of buttons. As you'll see, without it the button has a gray gradient background.